sample/src/samples/datepicker/advanced-options.js

```diff
@@ -1,3 +1,7 @@
+import {DatePickerDefaultParser} from '../../../../src/aurelia-materialize-bridge/datepicker/datepicker.default-parser';
```

Comment:
Why not `import {DatePickerDefaultParser} from 'aurelia-materialize-bridge/datepicker/datepicker.default-parser';`? Did that not work?